.github/workflows/release-please.yml

```
@@ -0,0 +1,24 @@
+    push:
+      branches:
+        - main
+  
+  permissions:
+    contents: write
+    pull-requests: write
+  
+  name: release-please
+  
+  jobs:
+    release-please:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)